katalog/gatekeeper/rules/constraints/kustomization.yaml

```diff
@@ -11,6 +11,7 @@ resources:
   - must_have_unique_ingresses.yml
   - must_have_liveness_probe.yml
   - must_have_readiness_probe.yml
+  - must_have_namespace_label_to_be_safely_deleted.yml
```

Comment:
I'm not sure about having this one ON by default. What do the rest think about it?

Reply:
I think that once is clear how to handle this, is a great added value to have this onboard by default. In the past happened to me to accidentally delete a namespace in a production environment and I whish there was a such feature :)